riotfile.py

```diff
@@ -2548,13 +2548,14 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         ),
         Venv(
             name="aws_lambda",
-            command="pytest {cmdargs} tests/contrib/aws_lambda",
+            command="pytest --no-ddtrace {cmdargs} tests/contrib/aws_lambda",
```

Comment:
Why do we need `--no-ddrace` here? Do we need to disable the pytest integration for more tests?

Reply:
There were some crashes during test teardown that originated in ddtrace. If similar behavior surfaces in other suites, I'll apply the same approach.